tests/integ/modin/series/test_str_accessor.py

```diff
@@ -276,6 +290,15 @@ def test_str_no_params(func):
     )
 
 
+@sql_count_checker(query_count=0)
+def test_str_isdigit_negative():
```

Comment:
Is this a regression? The above test with the "isdigit" parameter isn't XFAILed and has no fallbaack query count.

Reply:
Ooh good catch, I think I missed the `snowflake_query_compiler` implementation. Reverting this.